site/docs/components/button/accessibility.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Button
+layout: DetailComponent
+sidebar:
+  label: Accessibility
+data:
+  descriptionRef:
+    $ref: ./#/data/description
+---
+
+### Best practises
+
+#### When a Button has both an icon and a label
+
+- Pass `aria-hidden=”true”` to the icon so that its label is not announced by screen readers.
+
+#### When a Button has an icon only and no label
+
+- Pass an `aria-label` to Button that describes its purpose.
+
+### Keyboard interactions
+
+| Key                                  | Function                                                                                                                                                                                                                                                                |
+| ------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <kbd>Tab</kbd>                       | - If focus is above the button, Tab key press moves focus onto the button.<br/>- If the button is disabled and is set to receive focus, receives focus.<br/>- If the button has focus, Tab moves focus out of the button, to next focusable component in the tab order. |
+| <kbd>Tab</kbd> and <kbd>Shift</kbd>  | - If focused, Tab+Shift combo moves focus out of the component to the previous component in the tab order.                                                                                                                                                              |
+| <kbd>Enter</kbd> or <kbd>Space</kbd> | - If the button has focus, Enter or Space key press activates the button.<br/>- If the button is disabled and can receive focus, does nothing.                                                                                                                          |
```

Comment:
I think we use capitals for component name (other comment)
There's also a difference here of 'if focused' and 'if the button has focus' between tab and shift and enter/space - I think the wording should be aligned for clarity